frontend/src/mocks/Data.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
please, call this file 'movies.json', in the feature your app might have other mocked data structures

BTW, do you use it somewhere?

Reply:
Im not using it anywhere but i will change the filename